controller.go

```diff
@@ -372,6 +372,12 @@ func syncPV(pv *PV) {
 				}
 			} else {
 				// The PV must have been created with this ptr; leave it alone.
```

Comment:
@thockin @jsafrane I'm not sure I think this change is totally correct, actually.

This is the case where a PV points to a PVC which is bound elsewhere, and the PV in question was not bound by the controller.

So, that means that someone intentionally created the PV w/ the claim ref on it.

Do we really want to mark such a volume as available?  It doesn't seem quite 'available' to me, though it is available in the sense that it hasn't been bound.  Perhaps I'm overthinking it?